L1Trigger/TrackTrigger/python/TrackQualityParams_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+TrackQualityParams = cms.PSet(qualityAlgorithm = cms.string("GBDT"), #None, Cut, NN, GBDT
+                              ONNXmodel = cms.string("L1Trigger/TrackTrigger/data/TrackQualityModels/GBDT_default.onnx"),
+                              # !! TO BE UPDATED !! 
+                              # The ONNX model should be found at this path, if you want a local version of the model:
+                              # git clone https://github.com/Chriisbrown/L1Trigger-TrackTrigger.git L1Trigger/TrackTrigger/data
```

Comment:
should this be added to https://github.com/cms-data/L1Trigger-TrackTrigger?

Reply:
done (pending https://github.com/cms-sw/cmsdist/pull/6532)